test/Common/SupportedLanguagesTestAttribute.cs

```diff
@@ -46,7 +46,7 @@ public SqlInlineDataAttribute(params object[] args)
         public override IEnumerable<object[]> GetData(MethodInfo testMethod)
         {
             if (testMethod == null) { throw new ArgumentNullException(nameof(testMethod)); }
-            if (this.testData.Count == 0) { throw new EmptyException(nameof(testMethod)); }
+            if (this.testData.Count == 0) { throw new ArgumentNullException(nameof(testMethod)); }
```

Comment:
This isn't really accurate - could you just throw a general Exception and then add a message indicating what the actual issue is instead?